src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,36 @@
+import { NavLink } from 'react-router-dom';
+import cn from 'classnames';
+
+export const Navbar = () => {
+  return (
+    <nav
+      data-cy="nav"
+      className="navbar is-fixed-top has-shadow"
+      role="navigation"
+      aria-label="main navigation"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <NavLink
+            to="/"
+            className={({ isActive }) => cn('navbar-item', {
```

Comment:
To improve the readability of the code, it is better to transfer the definition of classes to separate variables and use the ready-made result. You can see an example of use on the website of the [classnames ](https://www.npmjs.com/package/classnames)package.